src/main/java/com/minecolonies/api/items/registry/ToolTypeEntry.java

```diff
@@ -0,0 +1,91 @@
+package com.minecolonies.api.items.registry;
+
+import com.minecolonies.api.items.ModToolTypes;
+import com.minecolonies.api.util.constant.IToolType;
+import net.minecraft.network.chat.Component;
+import net.minecraft.resources.ResourceLocation;
+import net.minecraft.world.item.ItemStack;
+import net.minecraftforge.registries.RegistryObject;
+
+import java.util.function.Function;
+import java.util.function.Predicate;
+
+public class ToolTypeEntry implements IToolType {
+    private final String name;
+    private final boolean variableMaterials;
+    private final Component displayName;
+    private final Predicate<ItemStack> isTool;
+    private final Function<ItemStack, Integer> itemLevel;
+
+    private ToolTypeEntry(final String name, final boolean variableMaterials, final Component displayName, final Predicate<ItemStack> isTool, final Function<ItemStack, Integer> itemLevel) {
+        this.name = name;
+        this.variableMaterials = variableMaterials;
+        this.displayName = displayName;
+        this.isTool = isTool;
+        this.itemLevel = itemLevel;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public boolean hasVariableMaterials() {
```

Comment:
Variable materials is no longer required.

Reply:
Removed.